packages/lesswrong/lib/collections/fieldChanges/collection.ts

```diff
@@ -0,0 +1,57 @@
+import { foreignKeyField } from '@/lib/utils/schemaUtils';
+import { createCollection } from '../../vulcan-lib/collections';
+import { addUniversalFields } from '@/lib/collectionUtils';
+import { DatabaseIndexSet } from '@/lib/utils/databaseIndexSet';
+
+const schema: SchemaType<"FieldChanges"> = {
```

Comment:
Probably should be in a separate file, since I think we'll want to have them in separate files for when we decouple schemas from collections more strongly.